debezium-server/debezium-server-pubsub/pom.xml

```diff
@@ -30,7 +30,17 @@
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-core</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>com.google.api</groupId>
+            <artifactId>gax</artifactId>
+            <version>1.56.0</version>
```

Comment:
Any reason for this seemingly old version? There's 2.8.1 as the latest?

Reply:
@nathan-smit-1k, any thoughts on that one?

Btw. we generally prefer rebases over merges (I'll take care of that upon integrating that PR).

Reply:
Hey @gunnarmorling .  Sorry about that, I did a fetch upstream and don't have separate branches (my fork is a bit disorganised).

I made a mistake on this.  I don't need to explicitly add these dependencies to the POM.xml file and it's probably a bad idea to have them there anyway so I've reverted the POM to its original state.  Both threetenbp and gax are actually dependencies of google-cloud-pubsub in maven so the version is related to the google-cloud-pubsub version which I believe is this one:  https://mvnrepository.com/artifact/com.google.cloud/google-cloud-pubsub/1.104.1 .  I've tested and everything compiles and runs fine

Reply:
Ok, makes sense. This LGTM now, will merge.